src/app/services/scroll.service.ts

```diff
@@ -38,11 +38,21 @@ export class ScrollService {
     @Inject(DOCUMENT) private document: any,
     private platform: PlatformService
   ) {
-    // provide observable with top / bottom vertical offset
+    // kill overscroll, primarily for safari so the map is zoomable with
+    // mousewheel / trackpad.
+    this.platform.nativeWindow.addEventListener('mousewheel', (e) => {
```

Comment:
Is this firing on every single event? Or only the first time?

Reply:
It's firing for every single one, because `preventDefault` needs to be called on all of them to prevent the overscroll from happening.  There's not much overhead going on in the handler so it shouldn't really effect performance, but I could do a user agent check and only set it for Safari.